pom.xml

```diff
@@ -86,6 +86,11 @@
     <dependencyManagement>
         <dependencies>
             <!-- overrides of imports -->
+            <dependency>
```

Comment:
to remove next powsybl upgrade?

Reply:
I will update with the released version of powsybl-ws-commons anyway, so I will ad the comment